config/localization/en/app.json

```diff
@@ -981,18 +981,21 @@
       "ORDER_FILLED": "Filled",
       "ORDER_LINES": "Order Lines",
       "ORDER_LINES_TOOLTIP": "Display or hide order lines",
-      "ORDER_REMOVAL_REASON_EXPIRED": "Expired",
-      "ORDER_REMOVAL_REASON_FOK_ORDER_COULD_NOT_BE_FULLY_FULLED": "Fill or Kill order could not be fully filled",
-      "ORDER_REMOVAL_REASON_IMMEDIATE_OR_CANCEL_WOULD_REST_ON_BOOK": "Immediate or Cancel order would rest on book",
-      "ORDER_REMOVAL_REASON_INDEXER_EXPIRED": "Internal error",
-      "ORDER_REMOVAL_REASON_INTERNAL_ERROR": "Internal error",
-      "ORDER_REMOVAL_REASON_POST_ONLY_WOULD_CROSS_MAKER_ORDER": "Post Only would match maker order",
-      "ORDER_REMOVAL_REASON_REDUCED_ONLY_RESIZE": "Reduced only resize",
-      "ORDER_REMOVAL_REASON_REPLACED": "Replaced",
-      "ORDER_REMOVAL_REASON_SELF_TRADE_ERROR": "Self trade error",
-      "ORDER_REMOVAL_REASON_UNDERCOLLATERALIZED": "Undercollateralized",
-      "ORDER_REMOVAL_REASON_UNSPECIFIED": "Unspecified",
-      "ORDER_REMOVAL_REASON_USER_CANCELED": "User canceled",
+      "ORDER_REMOVAL_REASON_EQUITY_TIER": "The subaccount does not satisfy the equity tier requirements.",
+      "ORDER_REMOVAL_REASON_EXPIRED": "The order has expired.",
+      "ORDER_REMOVAL_REASON_FOK_ORDER_COULD_NOT_BE_FULLY_FULLED": "The FOK order could not be fully filled.",
```

Comment:
nit: I think Fill or Kill is more clear to users, who might not know what FOK means